tests/rados/scheduled_scrub_scenarios.py

```diff
@@ -0,0 +1,82 @@
+"""
+   This module contains the methods required for scrubbing.
+   1.To set the parameters for scrubbing initially required the
+     cluster time and day details.get_cluster_date method provides
+     the details.
+   2.set_osd_configuration method  used to set the configuration
+     parameters on the cluster.
+   3.get_osd_configuration  method is used to get the configured parameters
+     on the cluster.
+     NOTE: With set_osd_configuration & get_osd_configuration methods can
+          use to set the get the any OSD configuration parameters.
+   4. get_pg_dump  method is used to get the pg dump details from the cluster.
+   5. verify_scrub  method used for the verification of scheduled scrub.
+"""
+
+import logging
+import time
+from datetime import datetime
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.rados_scrub import RadosScrubber
+
+log = logging.getLogger(__name__)
+
+
+def run(ceph_cluster, **kw):
+
+    OSD_SCRUB_MIN_INTERVAL = 1800
+    OSD_SCRUB_MAX_INTERVAL = 3600
+    OSD_DEEP_SCRUB_INTERVAL = 3600
+
+    log.info(run.__doc__)
+    config = kw["config"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    rados_obj = RadosScrubber(node=cephadm)
+
+    # Storing the pg dump log before setting the scrub parameters
+    before_scrub_log = rados_obj.get_pg_dump("pgid", "last_scrub_stamp")
+
+    # Get the cluster time
+    current_time = rados_obj.get_cluster_date()
+
+    # Preparation of cofiguration parameter values from the current
+    # cluster time
+    yy, mm, dd, hh, day = current_time.split(":")
+    date = datetime.datetime(int(yy), int(mm), int(dd), int(hh))
```

Comment:
I believe `date = datetime.datetime.now()` would help

Reply:
We required the current time of the container. The "datetime.datetime.now()" provide us the driver machine time. I notice that the timezones are different from the driver machine, installer node to the container. At Line-41, I am getting the container time.
More details at- https://bugzilla.redhat.com/show_bug.cgi?id=2023169

Reply:
As per the discussion I tried with the strptime, In this by default hh:mm:ss are getting added with 0 values. If that is the case again required additional lines in the file.
Code sample:
import os
from datetime import datetime

out=os.popen("date +%Y:%m:%d").read()
out=out.strip()
date_obj=datetime.strptime(out,"%Y:%m:%d")
print("Date object=",date_obj)

Output: Date object= 2021-11-18 00:00:00

Reply:
Okay.
During our discussion, I believed that we agreed to modify `get_cluster_date` to retrieve all the values instead of specifics. As below we are using `datetime.timedelta` which does not impact our logic below.

Since, `get_cluster_date` comes from a module, I believe we should retrieve the all the fields and allow calling methods to process/filter their requirement.

The current implementation is okay...